samples/force-sensitive-resistor/FsrWithAdcSample.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Iot.Device.Mcp3008;
+
+namespace force_sensitive_resistor
+{
+    class FsrWithAdcSample
+    {
+        private int _resistance = 10_000; // kOhm
+        private int _voltageSupplied = 3_300;  // 3300mV = 3.3V\
+        private Mcp3008 _adcConvertor;
+
+        public FsrWithAdcSample()
+        {
+            // Create a ADC convertor instance you are using depending how you wired ADC pins to controller
+            // in this example used ADC Mcp3008 with "bit-banging" wiring method.
+            // please refer https://github.com/dotnet/iot/tree/master/src/devices/Mcp3008/samples for more information
+            _adcConvertor = new Mcp3008(18, 23, 24, 25);
+        }
+
+        public int CalculateVoltage(int readValue)
```

Comment:
I think this should return double - especially because you divide by 1023

Reply:
ok